### PR TITLE
Rdiscrowd 1898

### PR DESCRIPTION
**Pull request by mariarivera30**
Wizard update to take in consideration the guidelines.
Also refactor wizard.py

### CODE REVIEW

**Comment by mariarivera30** [comment]
Screen Shot 2019-08-02 at 5 00 20 PM

This is how the wizard looks now

File: pybossa/wizard.py [hunk 1]
```diff
@@ -69,6 +60,15 @@ def task_presenter(self):
 
         return task_presenter
 
+    def task_guidelines(self):
+        if self.not_project_exist():
+            return False
+        guideline_editor_default_value = '<p><br></p>'
```
Comment (kbecker42) [inline pybossa/wizard.py]: Just curious if there was a reason for using a pre-defined HTML block for initializing or could this be left empty? :)

```python
guideline_editor_default_value = ''
```

File: test/test_wizard.py [hunk 2]
```diff
@@ -104,17 +112,20 @@ def test_wizard_get_href_project(self):
     def test_wizard_run_checks_project(self):
         project = self.get_project()
         project['info']['task_presenter'] = ''
+        # default value guidelines <p><br></p>
+        project['info']['task_guidelines'] = '<p><br></p>'
```
Comment (kbecker42) [inline test/test_wizard.py]: Similar to comment above, regarding the default value or an empty string. Thanks!

File: pybossa/wizard.py (hunk 1 above)
Comment (mariarivera30) [reply pybossa/wizard.py]: The editor produce that value  '<p><br></p>' if there is nothing. But actually I should check for '' and '<p><br></p>' assuming they click save with an empty editor.

A better way might be check if the value is equal that '<p><br></p>' then I keep it  as ''. Then I only need that default value at the saving.

**Pull request merged**